mmocr/models/textrecog/backbones/svtr.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Modified from <https://arxiv.org/abs/2205.00159>
+# Adapted from <https://github.com/PaddlePaddle/PaddleOCR>
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+from mmengine.model import BaseModule
+
+
+class OverlapPatchEmbed(BaseModule):
+    """Image to the progressive overlapping Patch Embedding.
+
+    Args:
+        input_size (int | tuple): The size of input, which will be used to
+            calculate the out size. Defaults to [32, 100].
+        in_channels (int): Number of input channels. Defaults to 3.
+        embed_dims (int): The dimensions of embedding. Defaults to 768.
+        num_layers (int, optional): Number of Conv_BN_Layer. Defaults to 2.
+    """
+
+    def __init__(self,
+                 img_size=[32, 100],
+                 in_channels=3,
+                 embed_dims=768,
+                 num_layers=2,
+                 init_cfg=None):
+
+        super().__init__(init_cfg=init_cfg)
+        """
+        num_patches = (img_size[1] // (2 ** num_layers)) * \
+                      (img_size[0] // (2 ** num_layers))
+        """
```

Comment:
Delete. BTW, triple quotes in python are used to denote docstrings instead of comments

Reply:
Because I need the object `num_patches` to construct the model in the end, I used the triple quotes to prevent forgetting.

Reply:
I'd suggest removing it if `img_size` is just used to verify the dimension irrelevant to a successful forward of this module. Modules are better implemented with no assumption on their runtime context and should not assert something outside their scope.